kmod/muser.c

```diff
@@ -1480,23 +1479,25 @@ static inline int mmap_done(struct mudev_cmd * const mucmd)
 	int ret;
 
 	if (cmd->err < 0)
-		return -1;
+		return -EINVAL;
+
 	ret = do_pin_pages(addr, mucmd->mmap_len, 1, &mucmd->pg_map);
 	if (ret) {
 		muser_alert("failed to pin pages: %d", ret);
 		mucmd->pg_map.pages = NULL;
 		mucmd->pg_map.nr_pages = 0;
 	}
+
 	return ret;
 }
 
-static long libmuser_unl_ioctl(struct file *filep,
-				     unsigned int cmd, unsigned long arg)
+static long libmuser_unl_ioctl(struct file *filep, unsigned int cmd,
+			       unsigned long arg)
```

Comment:
white space

Reply:
Its strange I do not see a white space in the file. Although the line is been adjusted to be started after the '('.
like: 
static long libmuser_unl_ioctl(struct file *filep, unsigned int cmd,
                                                  unsigned long arg)